apph-back/src/main/java/com/viseo/apph/domain/Setting.java

```diff
@@ -0,0 +1,33 @@
+package com.viseo.apph.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "setting")
+public class Setting extends BaseEntity {
+    long uploadSize;
+    long downloadSize;
```

Comment:
Je pense que ça vaudrait le coup que tu initialises ces valeurs du coup ?
```suggestion
    long uploadSize = 10 mb;
    long downloadSize = 15 mb;
```